packages/server/src/models/EventModel.ts

```diff
@@ -4,6 +4,8 @@ import BaseModel, { UuidType } from './BaseModel';
 
 export default class EventModel extends BaseModel<Event> {
 
+	private eventTtl_: number = 7 * 24 * 60 * 60 * 1000;
```

Comment:
`7 * Day` is clearer (import the Day constant)

Reply:
I was using TokenModel as a reference that has the exact same line. I changed it there as well.